pkg/reconciler/revision/resources/deploy_test.go

```diff
@@ -44,6 +42,8 @@ import (
 	"knative.dev/serving/pkg/networking"
 	"knative.dev/serving/pkg/queue"
 
+	_ "knative.dev/pkg/metrics/testing"
+	_ "knative.dev/pkg/system/testing"
```

Comment:
what do these imports do?

Reply:
Some test things. I just re-arranged them, as in I didn't add them.
But usually when I try to remove them things fail :)

Reply:
`"knative.dev/pkg/metrics/testing"` — actually avoids lots of error logs, but is harmless otherwise.